doc/dftb+/manual/dftbp.tex

```diff
@@ -458,6 +458,28 @@ \subsubsection{Optimiser: FIRE}
 \end{verbatim}
 
 
+\subsubsection{Optimiser: SteepestDescent}
+\label{sec:dftbp.Optimiser.SteepestDescent}
+
+Steepest descent optimiser, that proposes displacements according to the (scaled) gradient value.
+
+\begin{ptable}
+  \kw{ScalingFactor} & r &  & 1.0 & \\
```

Comment:
Coded as a different value
```suggestion
  \kw{ScalingFactor} & r &  & 0.1 & \\
```

Reply:
Yes, sorry for that. According to my testing (geoopt) a value of 1.0 performs quite well, so I would suggest to consistently switch it to 1.0 then. Let me fix that real quick.

Reply:
Should be consistently set to 1.0 now.